lib/travis/api/serialize/v2/http/user.rb

```diff
@@ -37,13 +37,23 @@ def user_data
                   'synced_at' => format_date(user.synced_at),
                   'correct_scopes' => Github::Oauth.correct_scopes?(user),
                   'created_at' => format_date(user.created_at),
+                  'first_logged_in_at' => format_date(user.first_logged_in_at),
+                  'intercom_user_hash' => intercom_user_hash,
                   'channels' => channels
                 }
               end
 
               def channels
                 ["private-user-#{user.id}"]
               end
+
+              def intercom_user_hash
+                OpenSSL::HMAC.hexdigest(
```

Comment:
And maybe we should wrap this in an `if ENV['USER_HASH_SECRET_KEY']`?
And change the `INTERCOME_SECRET_KEY` below to `USER_HASH_SECRET_KEY`?

Reply:
Do you think the property should be excluded entirely when `USER_HASH_SECRET_KEY` is absent? For enterprise and .org, etc?